experimental/dds/tree2/src/feature-libraries/editable-tree-2/proxies/proxies.ts

```diff
@@ -162,8 +171,30 @@ export function createObjectProxy<TSchema extends StructSchema, TTypes extends A
 				return undefined;
 			},
 			set(target, key, value) {
-				// TODO: Implement set
-				return false;
+				const fieldSchema = content.schema.structFields.get(key as FieldKey);
+
+				if (fieldSchema === undefined) {
+					// TODO: Verify that returning false will throw a TypeError in strict mode.
```

Comment:
The docs do claim this is true, but you're thinking it's worth a test?
Regardless, this made me wonder: will we ever run in sloppy mode? If so, should we just throw here?

Reply:
I added the missing test.  Thanks for the reminder.